server/util/body-parser.js

```diff
@@ -127,6 +128,24 @@ export function convertWpList(node) {
   }
 }
 
+function convertTweet(node) {
+  const className = node.attrs && getAttrVal(node.attrs, 'class');
+  const isTweet = Boolean(className && className.match('embed-twitter'));
+
+  if (!isTweet) return node;
+
+  const tweetHtml = node.childNodes.reduce((acc, cur) => {
```

Comment:
Yeah probably – I was thinking this was probably trying to be unnecessarily clever. Will change.